users/urls.py

```diff
@@ -1,13 +1,29 @@
 from django.conf.urls import url
-from django.contrib.auth.views import (login, logout, password_change,
-                                       password_change_done, password_reset,
-                                       password_reset_complete,
-                                       password_reset_confirm,
-                                       password_reset_done)
+from django.contrib.auth import views as auth_views
 
 from .views import (activate, activation_complete, register,
                     registration_closed, registration_complete)
 
+# Attempt to use the auth class based views if available.
+try:
+    login_view = auth_views.LoginView.as_view()
```

Comment:
AFAIK this won't work. e.g. `LoginView.as_view()` need to called w/ `template_name` as kwarg.

Same issue w/ all the views being called without kwags. 

I would prefer that we simply use the class based views.. and add a warning to README for folks to pin their requirements to use version 0.2.1 or earlier for older django.